rules/migrate-an-existing-user-store-to-an-externalauthprovider/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you know how to migrate an existing user store to an ExternalAuthProvider?
-uri: integrate-identityserver-with-an-existing-user-store
+uri: migrate-an-existing-user-store-to-an-externalauthprovider
```

Comment:
Just in case can you also add a redirect to the old name: `integrate-identityserver-with-an-existing-user-store`